archie-utils/src/main/java/com/nedap/archie/xml/JAXBUtil.java

```diff
@@ -44,4 +44,31 @@ private static synchronized void initArchieJaxbContext() {
             }
         }
     }
+
+    public static synchronized JAXBContext createAOMContext() {
+        try {
+            List<Class> classes = new ArrayList<>();
+            classes.addAll(ArchieAOMInfoLookup.getInstance().getRmTypeNameToClassMap().values());
+            //classes.addAll(ArchieRMInfoLookup.getInstance().getRmTypeNameToClassMap().values());
```

Comment:
This looks copy pasted and can be removed? Same in the createRMContext method below, but the other way around

Reply:
Well it can be useful to create a JAXBContext for only the RM and AOM. At least to generate the XSDs to check if they are same, but it has more uses. so I left them in, potentially will add a test for them later on.

Reply:
I'll remove the commented line, perhaps I'll add a method to abstract this a bit.